diff --git a/drivers/Kconfig b/drivers/Kconfig
--- a/drivers/Kconfig
+++ b/drivers/Kconfig
@@ -9,6 +9,7 @@
 source "drivers/cxl/Kconfig"
 source "drivers/pcmcia/Kconfig"
 source "drivers/rapidio/Kconfig"
+source "drivers/rknpu/Kconfig"
 
 
 source "drivers/base/Kconfig"
diff --git a/drivers/Makefile b/drivers/Makefile
--- a/drivers/Makefile
+++ b/drivers/Makefile
@@ -67,6 +67,8 @@
 # gpu/ comes after char for AGP vs DRM startup and after iommu
 obj-y				+= gpu/
 
+obj-$(CONFIG_ROCKCHIP_RKNPU)		+= rknpu/
+
 obj-$(CONFIG_CONNECTOR)		+= connector/
 
 # i810fb depends on char/agp/
